### PR TITLE
sitl-with-morse.rst:Updated the documentation related to Morse

### PR DESCRIPTION
**Pull request by arshPratap**
Updated the morse documentation to reflect the following changes:

- Added a note signifying that Morse is not supported(see [here](https://github.com/morse-simulator/morse/blob/master/README.md))

- Improvements for the installation section(see [here](https://github.com/morse-simulator/morse/issues/822) ) 

- Minor grammar fixes

Thanks!

### CODE REVIEW

File: dev/source/docs/sitl-with-morse.rst [hunk 1]
```diff
@@ -3,6 +3,9 @@
 =====================
 Using SITL with Morse
 =====================
+.. warning::
+
+    Morse robotics simulator is no longer supported!
```
Comment (khancyr) [inline dev/source/docs/sitl-with-morse.rst:8]: Maintained, we still support its backend on ArduPilot side

**Comment by rishabsingh3003** [comment]
Coincidently I was going to add a note for the Blender Version requirements when a couple of users asked me.. Can we make it a little bit more highlighted? It's something many people are doing wrong..

File: dev/source/docs/sitl-with-morse.rst (hunk 1 above)
Comment (arshPratap) [reply dev/source/docs/sitl-with-morse.rst]: Thanks for this @khancyr  sir.Got a little confused while reading the Morse Github. Nevertheless have made the changes

**Comment by arshPratap** [comment]
@rishabsingh3003 thanks for the note! I have made some changes to the Blender Version documentation in corresponding to the suggestions. Do take a look. Thanks!!

**Review by rishabsingh3003 — APPROVED** [review]
LGTM!

**Comment by Hwurzburg** [comment]
merging based on approval

**Pull request merged**